bindings/wasm/native/Cargo.toml

```diff
@@ -0,0 +1,38 @@
+[workspace]
+
+[package]
+name = "iota-client-wasm"
```

Comment:
For streams and iota.rs/nodejs we got rid of the iota prefix, as publishing them would already be @iota/client-wasm.

Its at least consistent then. Would require changes to the imports on all examples if you chose to

Reply:
Why would it be `@iota/client-wasm` when publishing it? The npm package maybe, but not the Rust crate?